xrpl/models/flags.py

```diff
@@ -54,6 +56,10 @@ def interface_to_flag_list(
             f[:-4]: _flag_enum_to_dict(getattr(transactions.pseudo_transactions, f))
             for f in pseudo_tx_flag_enums
         },
+        **{
```

Comment:
`interface_to_flag_list` appears to be used in conjunction with `Transaction` class. Why are we including `LedgerObject` flags in this list?